src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::unnecessary_filter_map)]
```

Comment:
Let's comment why we added this, please.

Reply:
Done!